content/it/01_friendly-introduction-to-zero-knowledge.md

```diff
@@ -0,0 +1,553 @@
+---
+
+title: 'Introduzione semplificata alla Zero Knowledge'
+date: '2023-07-17'
+tags: \['zero-knowledge']
+draft: false
+layout: PostSimple
+slug: "/it/friendly-introduction-to-zero-knowledge"
+images: \['../assets/01\_zkp-magic.png']
```

Comment:
No slash, see https://raw.githubusercontent.com/zkintro/zkintro/refs/heads/main/content/zh-tw/01_friendly-introduction-to-zero-knowledge.md